source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/veff_lcao.h

```diff
@@ -69,9 +71,10 @@ class Veff<OperatorLCAO<T>> : public OperatorLCAO<T>
 
     elecstate::Potential* pot = nullptr;
 
-    bool allocated_pvpR = false;
+    std::vector<ModuleBase::Vector3<double>> kvec_d;
```

Comment:
@haozhihan Why do you use a copy but not a pointer such as "ModuleBase::Vector3<double>* kvec_d"?

Reply:
@dyzheng Sorry, it should be `const std::vector<ModuleBase::Vector3<double>>& kvec_d`. I will pull requests to modify this inefficient operation.